docker-compose.yml

```diff
@@ -2,16 +2,16 @@ version: '2'
 
 services:
   frontend:
+    image: index.docker.io/buildkite/frontend
     build: .
-    volumes:
-      - "./bundle-analysis:/frontend/bundle-analysis"
-      - "./dist:/frontend/dist"
-      - "./coverage:/frontend/coverage"
     environment:
-      BUILDKITE:
-      BUILDKITE_COMMIT:
-      BUILDKITE_ORGANIZATION_SLUG:
-      BUILDKITE_PIPELINE_SLUG:
-      CI:
-      EMOJI_HOST:
-      FRONTEND_HOST:
+      - EMOJI_HOST
+      - FRONTEND_HOST
+      - BUILDKITE
+      - BUILDKITE_COMMIT
+      - BUILDKITE_ORGANIZATION_SLUG
+      - BUILDKITE_PIPELINE_SLUG
+      - CI
+    volumes:
+      - ".:/frontend:cached"
```

Comment:
The docker-compose plugin doesn't use ephemeral scripts, because it overrides the `command` hook (which is the thing that generates and uses those ephemeral scripts).